### PR TITLE
Updated based on internal feedback

### PR DESCRIPTION
**Pull request by dnarain-astro**

### CODE REVIEW

File: astro/cross-account-role-setup.md [hunk 1]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
-As the Astro service changes, Astronomer support might change the role policy attached to the cross-account role. Astronomer support informs all organizations by email before implementing any changes. To make sure you receive these notifications, subscribe to the [Status Page](https://status.astronomer.io/). 
-
-The following table lists the permissions required by Astro to manage the cross-account role.
-
-| Permissions                                                                               | Purpose                                                |
-| ----------------------------------------------------------------------------------------- | ----------------------------------------------------- |
-| `iam:CreatePolicy , iam:CreatePolicyVersion , iam:DeletePolicy , iam:DeletePolicyVersion` | Manage the policy attached to the cross-account role. |
-
-#### Monitor the cross-account role for changes (optional)
+When a data plane is activated on AWS, a cross-account IAM role and supporting policies are created. These define the set of actions the control plane is permitted to perform in the management of the data plane.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:77]:
```suggestion
When you activate a data plane on AWS, a cross-account IAM role and supporting policies are created to define the set of actions the control plane is permitted to perform in the management of the data plane.
```

File: astro/cross-account-role-setup.md [hunk 2]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
+
+Astronomer follows the principles of least privilege access and policies are defined to support the functionality of the entire Astro product. You can preview this role by examining [the CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml) that will be used to create this role.                                                                          |
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:79]:
```suggestion
Astronomer has implemented least privilege access policies to support Astro functionality. To preview the roles that support these policies, see the [CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml).                                                                          |
```

File: astro/cross-account-role-setup.md [hunk 3]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
+
+### Notification of changes to the cross account role
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:81]:
```suggestion
### Notification of changes to the cross-account role
```

File: astro/cross-account-role-setup.md [hunk 4]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
+
+From time-to-time, it may be necessary to make changes to these policies to ensure continued operation and evolution of the Astro platform.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:83]:
```suggestion
Occasionally, Astronomer will change its policies to ensure the continued operation and development of Astro. 
```

File: astro/cross-account-role-setup.md [hunk 5]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
+When a data plane is activated on AWS, a cross-account IAM role and supporting policies are created. These define the set of actions the control plane is permitted to perform in the management of the data plane.
+
+Astronomer follows the principles of least privilege access and policies are defined to support the functionality of the entire Astro product. You can preview this role by examining [the CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml) that will be used to create this role.                                                                          |
+
+### Notification of changes to the cross account role
+
+From time-to-time, it may be necessary to make changes to these policies to ensure continued operation and evolution of the Astro platform. 
+
+Astronomer will notify users with an Organization Owner role at least 14 days prior to any changes to a policy governing the cross-account IAM role that result in an *expansion* of access. The notification will be sent via email, reflect the change(s) to the policy, and include a brief explanation for the change.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:85]:
```suggestion
Fourteen days prior to a change to the policy governing the cross-account IAM role that results in an expansion of access, users with an Organization Owner role will receive a notification email from Astronomer support. The notification email describes the policy change an explanation of why it was necessary. 
```

File: astro/cross-account-role-setup.md [hunk 6]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
...
+
+Astronomer follows the principles of least privilege access and policies are defined to support the functionality of the entire Astro product. You can preview this role by examining [the CloudFormation template](https://astro-cross-account-role-template.s3.us-east-2.amazonaws.com/customer-account-prerelease.yaml) that will be used to create this role.                                                                          |
+
+### Notification of changes to the cross account role
+
+From time-to-time, it may be necessary to make changes to these policies to ensure continued operation and evolution of the Astro platform. 
+
+Astronomer will notify users with an Organization Owner role at least 14 days prior to any changes to a policy governing the cross-account IAM role that result in an *expansion* of access. The notification will be sent via email, reflect the change(s) to the policy, and include a brief explanation for the change. 
+
+Astronomer may *reduce* the access available to the cross-account role as part of standard change management approaches without any notifications.
```
Comment (kwcanuck) [inline astro/cross-account-role-setup.md:87]:
```suggestion
Astronomer can reduce the access available to the cross-account role without notification.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/cross-account-role-setup.md (hunk 3 above)
Comment (jwitz) [reply astro/cross-account-role-setup.md:81]: Maybe "Receive notification" to keep it action oriented?

File: astro/cross-account-role-setup.md [hunk 7]
```diff
@@ -74,44 +74,19 @@ To preview this role in its entirety, see its [CloudFormation template](https://
 
 ### Permissions used to manage data plane resources
 
-As the Astro service changes, Astronomer support might change the role policy attached to the cross-account role. Astronomer support informs all organizations by email before implementing any changes. To make sure you receive these notifications, subscribe to the [Status Page](https://status.astronomer.io/). 
-
-The following table lists the permissions required by Astro to manage the cross-account role.
-
-| Permissions                                           | Purpose                                                                                                                        |
```
Comment (jwitz) [inline astro/cross-account-role-setup.md:81]: @dnarain-astro Why have we decided to remove specific permission information from this page? Are the uses of each policy self-evident?

Comment (kwcanuck) [reply astro/cross-account-role-setup.md:81]: @jwitz this is the comment from @dnarain-astro in [PR 1541](https://github.com/astronomer/docs/pull/1541):

_Feedback has been that having each permission listed in this document is too AWS-centric (i.e. we are basically regurgitating what each API family does on AWS). Instead, we will point users to the policy in the public S3 bucket and they can examine this on their own._

**Comment by jwitz** [comment]
It looks like these changes were merged in https://github.com/astronomer/docs/pull/1541, so I'm going to close this PR out.

**Pull request closed without merging**